config/locales/views/dossier_mailer/notify_near_deletion_to_user/fr.yml

```
@@ -2,23 +2,23 @@ fr:
+        one: Un dossier traité va bientôt être supprimé
+        other: Des dossiers traités vont bientôt être supprimés
       header_en_construction:
-        one: "Afin de respecter la durée de conservation de vos données personnelles, le dossier en construction suivant sera supprimé automatiquement dans deux semaines :"
-        other: "Afin de respecter la durée de conservation de vos données personnelles, les dossiers en construction suivants seront supprimés automatiquement dans deux semaines :"
+        one: "Afin de respecter la durée de conservation de vos données personnelles, le dossier en attente d'instruction suivant sera supprimé automatiquement dans deux semaines :"
+        other: "Afin de respecter la durée de conservation de vos données personnelles, les dossiers en attente d'instruction suivants seront supprimés automatiquement dans deux semaines :"
       header_termine:
-        one: "Afin de respecter la durée de conservation de vos données personnelles, le dossier suivant dont le traitement est terminé sera supprimé automatiquement dans deux semaines :"
-        other: "Afin de respecter la durée de conservation de vos données personnelles, les dossiers suivants dont le traitement est terminé seront supprimés automatiquement dans deux semaines :"
+        one: "Afin de respecter la durée de conservation de vos données personnelles, ce dossier traité sera supprimé dans deux semaines :"
+        other: "Afin de respecter la durée de conservation de vos données personnelles, ces dossiers traités seront supprimés dans deux semaines :"
       account_active:
         one: Votre compte reste activé sur Démarches Simplifiées, seul le dossier sera supprimé.
         other: Votre compte reste activé sur Démarches Simplifiées, seuls les dossiers seront supprimés.
       footer_termine:
-        one: "Vous pouvez télécharger votre dossier au format PDF depuis l’onglet « Expirant » sur la page <a href=\"%{dossiers_url}\">Mes dossiers</a>."
-        other: "Vous pouvez télécharger vos dossiers au format PDF depuis l’onglet « Expirant » sur la page <a href=\"%{dossiers_url}\">Mes dossiers</a>."
+        one: Vous pouvez télécharger votre dossier au format PDF depuis l’onglet « Expirant » sur la page
+        other: "Vous pouvez télécharger vos dossiers au format PDF depuis l’onglet « Expirant » sur la page "
       footer_en_construction:
-        one: "Si vous souhaitez conserver votre dossier plus longtemps, vous pouvez <b>prolonger sa durée de conservation</b> dans l’interface."
-        other: "Si vous souhaitez conserver vos dossiers plus longtemps, vous pouvez <b>prolonger leur durée de conservation</b> au cas par cas dans l’interface."
+        one: Depuis la page de votre dossier vous avez la possibilité de :<br>- prolonger la durée de conservation pour un délai de 6 mois<br>- contacter l'administration qui gère votre dossier via la messagerie
```

C'est pas forcément 6 mois si ? ca serait pas plutot un truc du genre :

```suggestion
        one: Depuis la page de votre dossier vous avez la possibilité de :<br>- prolonger la durée de conservation pour un délai de %{procedure.duree_de_conservation_dans_ds} mois<br>- contacter l'administration qui gère votre dossier via la messagerie
```